asynchronous-authorization/langchain-next-js/package.json

```diff
@@ -25,7 +25,8 @@
     "node": ">=20 <23"
   },
   "dependencies": {
-    "@auth0/ai-langchain": "^3.5.0",
+    "@auth0/ai": "^5.0.0",
+    "@auth0/ai-langchain": "^4.0.0",
```

Comment:
nit: should be totally fine to just reference `"@auth0/ai-langchain": "^4.0.0",` now (since the other before was just while testing w/ temp packages/resolutions there).